src/interceptors/ClientRequest/NodeClientRequest.ts

```diff
@@ -115,21 +109,10 @@ export class NodeClientRequest extends ClientRequest {
     this.log('normalized arguments:', { chunk, encoding, callback })
 
     const requestBody = this.getRequestBody(chunk)
-    const isomorphicRequest = this.toIsomorphicRequest(requestBody)
-    const interactiveIsomorphicRequest: InteractiveIsomorphicRequest = {
-      ...isomorphicRequest,
-      respondWith: createLazyCallback({
-        maxCalls: 1,
-        maxCallsCallback() {
-          invariant(
-            false,
-            'Failed to respond to "%s %s" request: the "request" event has already been responded to.',
-            isomorphicRequest.method,
-            isomorphicRequest.url.href
-          )
-        },
-      }),
-    }
+    const request = this.toIsomorphicRequest(requestBody)
```

Comment:
I would keep `const isomorphicRequest` for clarity. There's always a `request` in the context, which is the intercepted request. Just so it doesn't get us confused.